tests/performance/3DLighting/attenuation/Environment.java

```diff
@@ -83,6 +83,9 @@
         pointLight2.setTranslateX(-LIGHT_X_DIST);
         spotLight2.setTranslateX(-LIGHT_X_DIST);
 
+        directionalLight1.setDirection(new Point3D(-LIGHT_X_DIST, 0, LIGHT_Z_DIST));
+        directionalLight2.setDirection(new Point3D(LIGHT_X_DIST, 0, LIGHT_Z_DIST));
```

Comment:
You need to add an import for `Point3D`.

Reply:
Weird that the IDE didn't warn. It seems like it did not identify the file as part of the source code.